token/services/certifier/interactive/client.go

```diff
@@ -98,95 +107,141 @@ func (d *CertificationClient) RequestCertification(ids ...*token.ID) error {
 		return nil
 	}
 
-	resultBoxed, err := d.viewManager.InitiateView(NewCertificationRequestView(d.channel, d.namespace, d.certifiers[0], toBeCertified...))
+	var resultBoxed interface{}
+	var err error
+	for i := 0; i < 3; i++ {
```

Comment:
Maybe we could have a constant for MaxAttempts or so

Reply:
yes